### PR TITLE
Mega menu: heading indentation and class name

### PR DESCRIPTION
**Pull request by virginiacc**
Indents headings in mega menu and updates heading/footer class names.

## Additions

- Adds css to indent headings in mega menu
- Updates nemo css to include indentation

## Testing

1. Check out branch
2. Run `gulp styles`
3. Verify that headings have same indentation as nav items in `Consumer Tools` vertical
4. Verify that headings/footer otherwise display as on production at desktop and mobile sizes

## Screenshots

screen shot 2017-08-23 at 7 42 31 am

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/unprocessed/css/organisms/mega-menu.less [hunk 1]
```diff
@@ -626,6 +626,10 @@ body {
                     border-left-color: @green;
                 }
             }
+
+            h3.h5 {
```
Comment (sebworks) [inline cfgov/unprocessed/css/organisms/mega-menu.less]: Is there a better way to target these elements?

File: cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css [hunk 2]
```diff
@@ -1069,6 +1069,7 @@ p+.m-global-banner_head,ul+.m-global-banner_head,ol+.m-global-banner_head,dl+.m-
     letter-spacing: 1px;
     line-height: 1.25;
     text-transform: uppercase;
+    padding-left: 1.42857143em;
```
Comment (sebworks) [inline cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css]: Please add a note about this is necessary.

Comment (virginiacc) [reply cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css]: Added in https://github.com/cfpb/cfgov-refresh/pull/3280/commits/7f0fedc0e7419fcf8505398b8a4499afffce6a61

File: cfgov/unprocessed/css/organisms/mega-menu.less (hunk 1 above)
Comment (virginiacc) [reply cfgov/unprocessed/css/organisms/mega-menu.less]: That's a good point! I updated the heading/footer class names in https://github.com/cfpb/cfgov-refresh/pull/3280/commits/7f0fedc0e7419fcf8505398b8a4499afffce6a61

File: cfgov/unprocessed/css/organisms/mega-menu.less [hunk 3]
```diff
@@ -462,6 +450,21 @@ body {
...
+
+            // add border between title and following list
+            &:not(&__hidden) + ul {
+                border-top: 1px solid @gray-40;
+            }
+        }
+
     } );
 
     // Desktop sizes.
```
Comment (Scotchester) [inline cfgov/unprocessed/css/organisms/mega-menu.less]: Missed an update to the `.nav_footer` class name below on L474.

Comment (virginiacc) [reply cfgov/unprocessed/css/organisms/mega-menu.less]: Thanks for catching that! Should be addressed in https://github.com/cfpb/cfgov-refresh/pull/3280/commits/08dcff0f6b54c7ab654a9e67d0fcb5ea126d9ddb

**Comment by Scotchester** [comment]
Fix confirmed :+1: Rebase or merge in from master once more and we're good to go.

**Pull request merged**